dell/src/test/java/org/apache/iceberg/dell/ecs/TestPropertiesSerDesUtil.java

```diff
@@ -20,8 +20,8 @@
 
 import java.util.Map;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
```

Comment:
please use AssertJ assertions. See also https://iceberg.apache.org/contribute/#assertj

Reply:
This class uses @classRule and can't be migrated without some significant refactoring which I feel need some more experience to do. Should I split this pull request into arrow and dell packages as arrow doesn't have any classRules?